include/Core/Users/Modules/DeleteUsersByUserMetaModule.php

```diff
@@ -25,17 +25,40 @@ protected function initialize() {
 		$this->action        = 'delete_users_by_meta';
 		$this->cron_hook     = 'do-bulk-delete-users-by-meta';
 		$this->scheduler_url = 'https://bulkwp.com/addons/scheduler-for-deleting-users-by-meta/?utm_source=wpadmin&utm_campaign=BulkDelete&utm_medium=buynow&utm_content=bd-u-ma';
-		$this->messages      = array(
-			'box_label'         => __( 'By User Meta', 'bulk-delete' ),
-			'scheduled'         => __( 'Users from with the selected user meta are scheduled for deletion.', 'bulk-delete' ),
-			'cron_label'        => __( 'Delete Users by User Meta', 'bulk-delete' ),
-			'confirm_deletion'  => __( 'Are you sure you want to delete all the users from the selected user meta?', 'bulk-delete' ),
-			'confirm_scheduled' => __( 'Are you sure you want to schedule deletion for all the users from the selected user meta?', 'bulk-delete' ),
-			/* translators: 1 Number of users deleted */
-			'deleted_one'       => __( 'Deleted %d user with the selected user meta', 'bulk-delete' ),
-			/* translators: 1 Number of users deleted */
-			'deleted_multiple'  => __( 'Deleted %d users with the selected user meta', 'bulk-delete' ),
-		);
+		$this->messages      = $this->get_messages();
+	}
+
+	/**
+	 * Returns array of messages for multisite or single site depending on the requirement.
+	 *
+	 * @return array Messages array.
+	 */
+	protected function get_messages() {
+		if ( is_multisite() ) {
+			return array(
+				'box_label'         => __( 'By User Meta', 'bulk-delete' ),
+				'scheduled'         => __( 'Users from with the selected user meta are scheduled for removal.', 'bulk-delete' ),
+				'cron_label'        => __( 'Remove Users by User Meta', 'bulk-delete' ),
+				'confirm_deletion'  => __( 'Are you sure you want to remove all the users from the selected user meta?', 'bulk-delete' ),
+				'confirm_scheduled' => __( 'Are you sure you want to schedule removal for all the users from the selected user meta?', 'bulk-delete' ),
+				/* translators: 1 Number of users removed */
+				'deleted_one'       => __( 'Removed %d user with the selected user meta', 'bulk-delete' ),
+				/* translators: 1 Number of users removed */
+				'deleted_multiple'  => __( 'Removed %d users with the selected user meta', 'bulk-delete' ),
+			);
+		} else {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method get_messages uses an else expression. Else is never necessary and you can simplify the code to work without else.](https://app.codacy.com/app/BulkWP/bulk-delete/pullRequest?prid=4259873)